src/dataHandler.py

```diff
@@ -60,4 +60,162 @@ def send_all_data(self, sendingTo, destination):
             writer = csv.writer(f)
             writer.writerow(data)
             f.close()
+
+    #gravitational acceleration 
+    gravity = 9.80665
+
+    #gas constant
+    gas_const = 8.3144598
+
+    #earth atmospheric molar mass
+    atmo_molar_mass = 0.289644
+
+    #vapourization heat of water
+    h2o_vapour_heat = 2501000
+
+    #specific gas constant of dry air
+    specif_gas_const_dry = 287
+    
+    #specific gas constant of water vapour
+    specif_gas_const_h2o = 461.5
+    
+    #specific heat of dry air
+    specif_heat_dryair = 1003.5
+
+    #initialized altitude at launch pad
+    initialized_altitude = 274.1
+    # just an estimate, actual data will be collected on site
+
+    #previous calculated height change
+    prev_height_change = 0
+    # previous change of height to compensate for Null case
+
+    #lapse rate of the air
+    lapse_rate = 0.0098
+    # this is the standard dry air lapse rate, altered by initializeLapseRate()
+
+
+    def initializeLapseRate(moist, press, temp):
```

Comment:
```suggestion
    def initialize_lapse_rate(moist, press, temp):
```

snake case function names for consistency!